### PR TITLE
Ripple and gatari mirrors

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: app/adapters/beatmap_mirrors/mirror_aggregate.py [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
 from app.adapters.beatmap_mirrors.nerinyan import NerinyanMirror
 from app.adapters.beatmap_mirrors.osu_direct import OsuDirectMirror
+from app.adapters.beatmap_mirrors.ripple import RippleMirror
 
 BEATMAP_MIRRORS: list[BeatmapMirror] = [
+    GatariMirror(),
     MinoMirror(),
     NerinyanMirror(),
     OsuDirectMirror(),
+    RippleMirror(),
```
Comment (cmyui) [inline app/adapters/beatmap_mirrors/mirror_aggregate.py]:
```suggestion
    # Disabled as ripple only supports ranked maps
    # RippleMirror(),
```

File: app/adapters/beatmap_mirrors/mirror_aggregate.py [hunk 2]
```diff
@@ -52,6 +58,9 @@ async def fetch_beatmap_zip_data(beatmapset_id: int) -> bytes | TimedOut | None:
     global_timeout = 15
     semaphore = asyncio.Semaphore(concurrency_limit)
 
+    # TODO: prioritization based on reliability, speed, etc.
+    random.shuffle(BEATMAP_MIRRORS)
```
Comment (cmyui) [inline app/adapters/beatmap_mirrors/mirror_aggregate.py:57]: Coming later in #3

**Review by tsunyoku — DISMISSED** [review]

File: app/adapters/beatmap_mirrors/gatari.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import logging
+
+from app.adapters.beatmap_mirrors import BeatmapMirror
+
+
+class GatariMirror(BeatmapMirror):
+    base_url = "https://osu.gatari.pw"
```
Comment (cmyui) [inline app/adapters/beatmap_mirrors/gatari.py:8]:
```suggestion
    name = "gatari"
    base_url = "https://osu.gatari.pw"
```

File: app/adapters/beatmap_mirrors/ripple.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import logging
+
+from app.adapters.beatmap_mirrors import BeatmapMirror
+
+
+class RippleMirror(BeatmapMirror):
```
Comment (cmyui) [inline app/adapters/beatmap_mirrors/ripple.py:6]:
```suggestion
class RippleMirror(BeatmapMirror):
    name = "ripple"
```

**Pull request merged**